google/cloud/firestore_v1/base_query.py

```diff
@@ -171,7 +186,7 @@ def _client(self):
         """
         return self._parent._client
 
-    def select(self, field_paths):
+    def select(self, field_paths) -> "BaseQuery":
```

Comment:
What is the significance of the quotes around `"BaseQuery"`?

Reply:
you can't specify the self type as far as I can tell other ways